buildpipeline/pipeline.json

```diff
@@ -83,9 +83,9 @@
           "Name": "DotNet-CoreFx-Trusted-Linux",
           "Parameters": {
             "PB_DockerTag": "rhel7_prereqs_2",
-            "PB_TargetQueue": "Redhat.72.Amd64",
             "PB_BuildArguments": "-buildArch=x64 -Release -portableLinux",
-            "PB_SyncArguments": "-p -portableLinux -- /p:ArchGroup=x64"
+            "PB_SyncArguments": "-p -portableLinux -- /p:ArchGroup=x64",
+            "PB_CreateHelixArguments": "/p:ArchGroup=x64 /p:ConfigurationGroup=Release /p:\"EnableCloudTest=true\" /p:\"TestProduct=corefx /p:\"TimeoutInSeconds=1200\" /p:\"TargetOS=Linux\" /p:\"TargetQueues=Redhat.72.Amd64;Debian.82.Amd64\"",
```

Comment:
If this is building on Linux, use commas... pretty sure the ; will end the line.